### PR TITLE
8 Add command groups access conditions

### PR DESCRIPTION
**Pull request by lbureau-billettiqueservices**
Issue : https://github.com/lbureau-billettiqueservices/GT4_support_schema/issues/8

Change key_indexes and access_conditions to more detailed cmd_group.

### CODE REVIEW

**Comment by lbureau-billettiqueservices** [comment]
Sorry for all the notifications, got surprised by the CI I set up myself. :)

This is ready for review and merge.

**Review by Spirtech-GT4 — APPROVED** [review]
Seems Ok to me.

**Comment by Spirtech-GT4** [comment]
Afterthoughts: is the presence of `cmd_group` optional (as, for example, `ef_type`)?
It should be optional, so that the truth is always in `fcp` (which is mandatory) rather than in its individual components.

**Comment by lbureau-billettiqueservices** [comment]
> Afterthoughts: is the presence of `cmd_group` optional (as, for example, `ef_type`)? It should be optional, so that the truth is always in `fcp` (which is mandatory) rather than in its individual components.

It is indeed an optional field, only FCP is mandatory.

File: schema/ticketing_support_schema.json [hunk 1]
```diff
@@ -59,11 +59,11 @@
       "type": "string",
       "pattern": "^[0-9a-fA-F]{2}$"
     },
-    "access_conditions": {
```
Comment (gcouenon) [inline schema/ticketing_support_schema.json:62]: my understanding was that access_conditions/key_indexes in a 4 byte format were kept in the schema (as optional). That said, I'm happy to keep only the detailed format in the schema.

**Review by gcouenon — APPROVED** [review]

File: schema/ticketing_support_schema.json (hunk 1 above)
Comment (lbureau-billettiqueservices) [reply schema/ticketing_support_schema.json:62]: I think our latest discussions effectively said that we could remove it. the FCP is enough.

**Comment by lbureau-billettiqueservices** [comment]
I realized i was missing cmd_group for EFs, another round of review please!

**Review by Spirtech-GT4 — APPROVED** [review]
Thanks (we all missed it...).

**Review by gcouenon — APPROVED** [review]
all good with me, thanks!

**Pull request merged**